package.json

```diff
@@ -13,6 +13,7 @@
     "package-term": "pnpm --filter=@gravitational/teleterm package",
     "storybook": "./web/scripts/run-storybook.sh",
     "storybook-smoke-test": "storybook dev -p 9002 -c web/.storybook --ci --smoke-test",
+    "test-storybook": "NODE_TLS_REJECT_UNAUTHORIZED=0 test-storybook -c web/.storybook --url https://localhost:9002 --skipTags=skip-test",
```

Comment:
When ran against `pnpm storybook`, then yeah, it's super slow since Vite runs in dev mode where each story and each file each story imports is individually compiled from scratch. After building a storybook that can be deployed, on my device the time to execute all tests went from 170s to 37s. `pnpm exec storybook build` takes around 18s.

```
# This builds the storybook to storybook-static.
pnpm exec storybook build --test -c web/.storybook

# Then we need to serve those static assets.
npx http-server storybook-static --port 9003 --silent

# And finally we can execute the tests against the built storybook.
pnpm exec test-storybook -c web/.storybook --url http://localhost:9003 --skipTags=skip-test
```

I based this on the [Run against non-deployed Storybooks](https://storybook.js.org/docs/writing-tests/test-runner#run-against-non-deployed-storybooks) section of docs.